examples/utils/bundle-size-tests/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@fluid-example/bundle-size-tests",
+  "version": "0.28.0",
+  "description": "A package for understanding the bundle size of Fluid Framework",
+  "homepage": "https://fluidframework.com",
+  "repository": "microsoft/FluidFramework",
+  "license": "MIT",
+  "author": "Microsoft",
+  "main": "dist/index.js",
```

Comment:
any idea if this will cause problems since this file doesn't exist?  looks like you covered the multiple entry points in the webpack config already and i don't think @fluid-example projects are published anywhere so maybe doesn't matter

Reply:
This packages shouldn't be consumed externally, but I'll remove this stuff. good catch